src/index.js

```diff
@@ -1,8 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+// import React from 'react';
+// import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+// ReactDOM.render(<App />, document.getElementById('root'));
+
+import { createRoot } from 'react-dom/client';
+const container = document.getElementById('root');
+const root = createRoot(container); // createRoot(container!) if you use TypeScript
+root.render(<App tab='home' />);
```

Comment:
This is cool! I'm curious, what does this do?

Reply:
I noticed that in DevTools console I was getting an error "ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it’s running React 17." 
Idk if everyone had it or it might be because I updated all my packages to latest version, but my settings for ReactDom were no longer compatible. 
While I was googling how to solve that issue I came across the website that I used as a giddiness, which solved the problem and warning dissapeared from console. Here is the link: https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis